src/org/zaproxy/zap/ZAP.java

```diff
@@ -87,7 +88,9 @@ public static void main(String[] args) throws Exception {
         } catch (final Exception e) {
         	// Cant use the CommandLine help here as the 
         	// i18n messages wont have been loaded
-            System.out.println("Failed due to invalid parameters. Use '-h' for more details.");
+            System.out.println("Failed due to invalid parameters: " + Arrays.toString(args));
+            e.printStackTrace();

```

Comment:
Does the printStackTrace() actually provide anything useful? I cant see how it will help users (or us) and I think it might end up just looking really messy. Stack traces are for working out where an error is, this is just the user supplying an invalid cmdline option..